CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing with NovusCore
+
+Awesome! You want to help and contribute to NovusCore!
+
+You can contribute in different ways for NovusCore, either by opening bug reports, suggesting fixes for already open issues, proposing fixes 
```

Comment:
There are several ways to contribute to NovusCore. You can help by suggesting new ideas, opening bug reports, test bug reports and submit fixes for already open issues by creating a pull request (PR).